examples/stable_diffusion_xl/configs/training/sd_xl_base_finetune_dreambooth_lora_910b.yaml

```diff
@@ -133,19 +133,19 @@ optim:
 
 data:
     per_batch_size: 1
-    total_step: 12000
-    num_parallel_workers: 6
+    total_step: 8000
```

Comment:
batch size=1 is too small for good optimization. Suggest to improve via gradient accumulation or stronger recompute or parallel training